src/components/shared/DropdownCustom.js

```diff
@@ -37,18 +36,16 @@ const DropdownCustom = (props) => {
 
       switch (e.target.textContent) {
         case 'Release date':
-          dispatch(setSort('release_date'));
-          dispatch(sortRelease(moviesByCriteria));
+          sortBy('release_date');
           break;
         case 'Rating':
-          dispatch(setSort('vote_average'));
-          dispatch(sortRating(moviesByCriteria));
+          sortBy('vote_average');
```

Comment:
Usually this check would live directly in the action:
```
sortBy(e.target.textContent)
```

Reply:
fix